Sources/SecureStorage.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  Keychain Property Wrapper.swift
+//  Due Tomorrow
+//
+//  Created by Morris Richman on 3/9/25.
+//
+
+import Foundation
+import KeychainSwift
+import SwiftUI
+
+/// Update the values to change the defaults for SecureStorage
+public class SecureStorageDefaults {
+    /// The `KeychainSwift` being saved with
+    public static var keychain = KeychainSwift()
+    /// Access controls for keychain
+    public static var access: KeychainSwiftAccessOptions? = nil
+}
+
+/// A property wrapper to easily and securely save data in Keychain
+@propertyWrapper
+public struct SecureStorage<T: ExpressibleByNilLiteral> {
+    /// The key for the value in keychain
+    let key: String
+    /// The `KeychainSwift` being saved with
+    var keychain: KeychainSwift
+    /// Access controls for keychain
+    var access: KeychainSwiftAccessOptions?
+    
+    /// The underlying get function for wrappedValue
+    /// - Returns: `T`
+    let _get: () -> T
+    /// The underlying set function for wrappedValue
+    /// - Parameters:
+    ///   - `T`: The new value being set
+    let _set: (T) -> Void
+    
+    public var wrappedValue: T {
+        get {
+            _get()
+        }
+        set {
+            _set(newValue)
+        }
+    }
+    
+    /// The Initializer for `SecureStorage`
+    /// - Parameters:
+    ///   - key: The key for the value in keychain
+    ///   - keychain: The `KeychainSwift` being saved with
+    ///   - access: Access controls for keychain
+    public init(
```

Comment:
If a type is unsupported, it might be a good idea to make the default initializer `internal` so users are notified at compile-time.